tableau_utilities/scripts/connection.py

```diff
@@ -136,9 +136,7 @@ def connection(args, server=None):
     """
 
     # Set Args
-    datasource_id = args.id if args.id else server.get_datasource(
-        datasource_name=args.name, datasource_project=args.project_name
-    ).id
+    datasource_id = args.id
```

Comment:
@JustinGrilli  This line crashes when updating a local file. In that case there's no need to specific a `project_name` or `name` 

```
 op run --env-file="/Users/jayrosenthal/.prod_env"  -- tableau_utilities --location local --file '/Users/jayrosenthal/Downloads/change_these_creds.tdsx' connection --connection_operation update_local_conn
ection --conn os_env
Traceback (most recent call last):
  File "/Users/jayrosenthal/.pyenv/versions/3.10.3/bin/tableau_utilities", line 33, in <module>
LOCAL OR SERVER: Local Operations
    sys.exit(load_entry_point('tableau-utilities==2.0.0', 'console_scripts', 'tableau_utilities')())
  File "/Users/jayrosenthal/.pyenv/versions/3.10.3/lib/python3.10/site-packages/tableau_utilities/scripts/cli.py", line 361, in main
    args.func(args)
  File "/Users/jayrosenthal/.pyenv/versions/3.10.3/lib/python3.10/site-packages/tableau_utilities/scripts/connection.py", line 139, in connection
    datasource_id = args.id if args.id else server.get_datasource(
AttributeError: 'NoneType' object has no attribute 'get_datasource'
```

I'm moving this call into the online function.